lesson2/NewProject/lib/src/main/java/com/example/MoneySaver.java

```diff
@@ -22,11 +22,12 @@ public Account getMyAccount() {
     }
 
     public String toString() {
-        return name + ", ";
+        return name + ", " + "My balance is: "+ myMoney;
```

Comment:
This was because you used an older version of the readme, I added the correct toString here